public/data/WilliamJ.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "William J Priest",
+    "type": "personal",
+    "bio": "!% Better Everyday",
+    "avatar": "https://github.com/WilliamJPriest.png",
+    "links": [
+      {
+        "name": "Code with me on GitHub",
+        "url": "https://github.com/WilliamJPriest",
+        "icon": "github"
+      },
+      {
+        "name": "laugh and learn with me on twitter" ,
+        "url": "https://twitter.com/WilliamJPriest1",
+        "icon": "twitter"
+      },
+      {
+        "name": "Job Hunt with me on Linkedin",
+        "url": "https://www.linkedin.com/in/william-j-priest-5aa7341a2/",
```

Comment:
Hey 👋 William,
The 👨‍💻 code looks good 🌠, 
but your LinkedIn URL is broken 💔.

Reply:
seems to be working when I click on it

Reply:
> seems to be working when I click on it

![image](https://user-images.githubusercontent.com/44284877/173755887-27ae2ece-21da-437e-9089-b66a3e9dc0e3.png)

maybe you are having private account, or some another issues

-> Go to profile:
- Edit your public profile & url:
  - https://www.linkedin.com/public-profile/settings?trk=d_flagship3_profile_self_view_public_profile
  - create a new user name - (if you want to remove numbres from it)
  
 or you can check setting page and explore

Reply:
you can check profile visibility 

![image](https://user-images.githubusercontent.com/44284877/173756660-37fd15d3-8504-4c66-8860-c514d6b03216.png)

Reply:
Awesome it worked, thank you